core/src/main/java/org/mobilitydata/gtfsvalidator/table/GtfsTableContainer.java

```diff
@@ -61,6 +71,13 @@ public boolean hasInvalidHeaders() {
         return invalidHeaders;
     }
 
+    /**
+     * Tells if the file is invalid headers, e.g., some required headers are missing.
+     * <p>
+     * Note that unknown headers are not considered invalid.
+     *
+     * @return true if the file has invalid headers, false otherwise
+     */
```

Comment:
Shouldn't that be `setMissingRequiredHeaders` ?
Honnestly I was confused only reading the documentation as I couldn't figure what could be invalid if unkown is ok. Then I reread.

Reply:
There may be other errors related to headers, e.g. the same column name used multiple times. Such files are rejected and further validation is not performed.